src/components/tile/__tests__/tile-test.js

```diff
@@ -0,0 +1,19 @@
+/** @jsx React.DOM */
+
+jest.dontMock('../code/views/tileView.jsx');
+
+var React = require('react/addons');
+var TileComponent = require('../code/views/tileView.jsx');
+var TestUtils = React.addons.TestUtils;
+
+describe('TileComponent', function() {
+
+  it('is an element', function() {
+    var img = {
+      src: 'foo',
+      alt: 'bar'
+    };
+    expect(TestUtils.isElement(<TileComponent image={img} title='title' />)).toBeTruthy();
```

Comment:
Before merge this should be refactored to use [react.addons.TestUtils](https://facebook.github.io/react/docs/test-utils.html#iselement) to test if it's the element/components we're expecting. Here's an example of some tests that utilize the `TestUtils` methods:

https://github.com/joestump/laminate/blob/master/src/components/__tests__/badge-tests.js